### ISSUE TITLE
A11y_SQLAzureDataStudio_DeployDacpac_ReviewDeployPlan_ScreenReader: Voiceover annouces the full table data in one navigation.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9264]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS1.3.1;#VoiceOver;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Screen reader: Voiceover
Mac OS:10.15.3
MAS References:MAS1.3.1

**Repro Steps**:
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select first radio button.
9. Navigate till next button & hit enter.
10. Provide all the details for Deploy Dacpac & hit next.
11. Launch Voiceover and navigate using control+option(left/right) arrow key.

**Actual:**
Issue1:
On navigating using the keys control+option (left/right) arrow key to the table all the data is announced in that navigation.
There is no proper row and column announcement or table name control.

Issue2:
Alert is announced on second navigation and on the third it announces the end of the line.

**Expected:**
Issue1:
The data should be announced when navigated to with proper table structure row and column announcements.

Issue2:
The alert should not be announced and instead of the end of the line out of the table should be announced.

**User Impact:**
Screen readers users will receive the information that the data is in a table structure.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EUO-vzIE47NOtZUDXycLaVcBW8CNmj3r40OIW0dlcSlQHQ?e=hcx7tc

**Comment by alanrenmsft** [comment]
dupe of #9266

**Comment by BhanuPrakashGit** [comment]
Verified the bug and issue does not repro.
**GithubTags:**#Closed;
PFA for reference

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
